R/model.R

```diff
@@ -308,7 +308,8 @@ tests.sleuth <- function(obj, lrt = TRUE, wt = TRUE) {
 #' results_table <- sleuth_results(sleuth_obj, 'conditionIP')
 #' @export
 sleuth_results <- function(obj, test, test_type = 'wt',
-  which_model = 'full', rename_cols = TRUE, show_all = TRUE) {
+  which_model = 'full', rename_cols = TRUE, show_all = TRUE,
+  aggregate_pval = obj$gene_aggregate) {
```

Comment:
Change obj$gene_aggregate to obj$pval_aggregate?

Reply:
Mirror pval_aggregate and aggregate_pval

Reply:
On the question --> yes.
And probably not expose `aggregate_pval` as an external argument; just do internal check (`obj$pval_aggregate `and `obj$gene_mode` should not both be `TRUE`, but ok if both are `FALSE`). That should just be set with `sleuth_prep` and then the user doesn't have to think about it again after that. The internal check will make sure that the users didn't inappropriately mess with those booleans between `sleuth_prep` and `sleuth_results`.